### PR TITLE
create new home with suggestion to logo sibs using react particles

### PR DESCRIPTION
**Pull request by tgmarinho**
Logo Suggestions to Home Page

- [x] Add 3 news sibelius logo
- [x] The colors of particles change randomly with each refresh on the page
- [x] The background is transparent, but we can use colors, it's in the comment

https://user-images.githubusercontent.com/380327/161402876-4cd6f90f-99c8-4b38-8275-2a9a444aa104.mp4

https://user-images.githubusercontent.com/380327/161402889-ebda940f-0b0f-4156-a0d4-4767c6a2af29.mp4

___

**NEW CHANGES ON LAYOUT**

- [x] add particles with sib's logo to home
- [x] timline with background transparent and zIndex = 1
- [x] add box shadow yellow to Menu Card  

https://user-images.githubusercontent.com/380327/161406769-7a735fc8-bc29-4018-bf85-e7ca5891239a.mp4

### CODE REVIEW

**Review by biantris — APPROVED** [review]
interesting o/

**Comment by tgmarinho** [comment]
lib https://malerba118.github.io/react-particle-image-demo/

**Review by Nick-Gabe — APPROVED** [review]

File: apps/web/src/pages/index.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const HomePage: NextPage<Props> = (props: Props) => {
         gap="12px"
         flexWrap="wrap"
       >
-        <Home />
+        <Home2 />
```
Comment (sibelius) [inline apps/web/src/pages/index.tsx:26]: Maybe we can just change the avatar instead of changing the whole home

**Review by sibelius — APPROVED** [review]

**Comment by tgmarinho** [comment]
Screen Shot 2022-04-02 at 19 40 17

**Comment by tgmarinho** [comment]
**NEW CHANGES ON LAYOUT**

- [x] add particles with sib's logo to home
- [x] timline with background transparent and zIndex = 1
- [x] add box shadow yellow to Menu Card  

https://user-images.githubusercontent.com/380327/161406769-7a735fc8-bc29-4018-bf85-e7ca5891239a.mp4

**Comment by tgmarinho** [comment]
compare
Screen Shot 2022-04-02 at 21 13 19

**Review by biantris — APPROVED** [review]

**Comment by Nick-Gabe** [comment]
I really liked both the logo and background, but I have some points:

- The yellow/green shadow doesn't fit in the context, some options would be:
  - using it as our primary or secondary color, but the blue would need to be replaced.
  - using a more "blue" color instead, to match with the actual palette.

- I don't like the timeline having no background, it feels too "loose in the enviroment", a black background with some opacity would solve this

But it is a great feature, having an animated bg like this is really cool!

**Pull request merged**